docs/Wiki/development/repositories/northstarmods.md

```diff
@@ -6,24 +6,51 @@ description: Core squirrel mods
 
 TODO
 
-When adding new Sqirrel script source files that are from the base game, make sure to commit the unmodified file first.
-In particular, on `main` they should committed as `Respawn` with the email address `respawn@northstar.tf`.
+## Reviewing PRs committing vanilla files
+
+When a PR adds new Squirrel script source files that are from the base game, make sure to commit unmodified files first to `main`.
+
+### 1. Update the PR branch
+
+Before adding vanilla files to `main`, make sure that the corresponding PR is up-to-date.
+Adding the vanilla file will cause a merge conflict on the PR, that is expected.
+
+### 2. Add vanilla file to NorthstarMods' `main`
+
+Before adding the vanilla file to `NorthstarMods`, make sure its end-of-line sequence is set to LF, as CRLF would cause merge conflicts on every single line.
```

Comment:
LF on Linux, CRLF on Windows. Basically using host native line endings.

Using non-host-native line endings is what causes issues.

(Internally git used LF)